h/util/markdown.py

```diff
@@ -125,6 +95,13 @@ def _linkify_rel(attrs, new=False):  # pylint: disable=unused-argument
     return attrs
 
 
+ALLOWED_TAGS = set(bleach.ALLOWED_TAGS + MARKDOWN_TAGS)
+
+MARKDOWN_ATTRIBUTES = {"a": _filter_link_attributes, "img": ["alt", "src", "title"]}
+ALLOWED_ATTRIBUTES = bleach.ALLOWED_ATTRIBUTES.copy()
+ALLOWED_ATTRIBUTES.update(MARKDOWN_ATTRIBUTES)
+
```

Comment:
This stuff could all go inside this function as private variables, or just be inlined.

Reply:
:+1: